### ISSUE TITLE
pivotal-ui.scss index file for all components

### ISSUE THREAD

**Issue by pivotal-gabriel-dumitrescu** [pivotal-cf/pivotal-ui#173]
We would like to import the Pivotal UI kit in one of our internal projects and use `@extend` to combine several classes instead of writing them explicitly in html:

```SCSS
    .apps-list {
      @extend .list-unstyled;
      @extend .type-neutral-11;
    }
```
and `<ul class="list-unstyled .type-neutral-11"></ul>` becomes `<ul class="apps-list"></ul>` 

Please include in the pivotal-ui source code and distribution, a file `pivotal-ui.scss` under `src/pivotal-ui` or root folder for distributions. 
It will contain a reference to all components developed in Pivotal UI. 
Source code version 1.6.1  used to have this file, but latest version does not.

It will look something like this: 
```SCSS
@import "components/alerts/alerts";
@import "components/alignment/alignment";
@import "components/avatars/avatars";
@import "components/back-to-top/back-to-top";
@import "components/backgrounds/backgrounds";
@import "components/bootstrap/bootstrap";
@import "components/buttons/buttons";
@import "components/code/code";
@import "components/collapse/collapse";
@import "components/colors/colors";
@import "components/deprecated/deprecated";
@import "components/dividers/dividers";
@import "components/dropdowns/dropdowns";
@import "components/ellipsis/ellipsis";
@import "components/embeds/embeds";
@import "components/expander/expander";
@import "components/forms/forms";
@import "components/google-maps/google-maps";
@import "components/grids/grids";
@import "components/health_indicators/health_indicators";
@import "components/hiring/product_designer";
@import "components/hoverable/hoverable";
@import "components/iconography/iconography";
@import "components/images/images";
@import "components/intro/intro";
@import "components/labels/labels";
@import "components/links/links";
@import "components/lists/lists";
@import "components/media/media";
@import "components/modals/modals";
@import "components/panels/panels";
@import "components/panes/panes";
@import "components/pivnet_homepage/pivnet_homepage";
@import "components/portals/portals";
@import "components/product_specific/product_specific";
@import "components/progress-bars/progress-bars";
@import "components/react-animations/react-animations";
@import "components/ribbons/ribbons";
@import "components/scales/scales";
@import "components/spinners/spinners";
@import "components/tables/tables";
@import "components/tabs/tabs";
@import "components/toggles/toggles";
@import "components/tooltips/tooltips";
@import "components/traffic_lights/traffic_lights";
@import "components/typography/typography";
@import "components/utils/utils";
@import "components/vertical-alignment/vertical-alignment";
@import "components/whitespace/whitespace";
```
Thanks

**Comment by stubbornella** [comment]
It looks like you are trying to do a Suzy framework style extends pattern. Pivotal UI isn't designed to be used like that. After experimenting with Sass extends, we found: 

1. It made the HTML very hard to understand, everything is a one-off and you can't guess what is happening from looking at the classes applied. 
1. Often we were creating objects that had too much responsibility. For example, grids that were also setting colors and adding margins. In short, a recipe for bugs. 
1. Or, HTML structure wasn't respected. Something would be made a grid column, but it's parent would not be made into a grid row. Very difficult to debug those kinds of errors because you can't easily discern parent-child relationships from the CSS and you can't see it in the html with Suzy-style extends.

In short, your first example is the right way to consume PUI:

```
<ul class="list-unstyled type-neutral-11"></ul>
```
I'm happy to have a longer chat about this. Ping me on slack if that sounds helpful.
